supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -8,6 +8,16 @@
 
 *See also*: xref:paas[PaaS], xref:saas[SaaS]
 
+[[ibm-cloud-bare-metal]]
+==== image:images/yes.png[yes] IBM Cloud® Bare Metal (Classic)
```

Comment:
Missing "(noun)"

Reply:
Thanks. I keep forgetting about this part. The construction will now display as `IBM Cloud® Bare Metal (Classic) (noun)`. Are the two sets of parentheses OK?

Reply:
I think it's not great, but I don't think there's anything we can do about it, if the the item in parentheses is part of the term. I think we have at least one other entry that does this as well.